imports/ui/case-wizard/case-wizard.jsx

```diff
@@ -115,6 +115,7 @@ class CaseWizard extends Component {
     const { mandatory, optional } = inputValues
     const { title, details, assignedUnitRole } = mandatory
     const { category, subCategory } = optional
+    const defaultValue = unitItem && unitItem.components.find(({default_assigned_to: assignedTo}) => assignedTo === userBzLogin).name
```

Comment:
You should move this into the `componentDidUpdate` lifecycle hook:
Check if unitItem is defined in contrast to `prevProps.unitItem` being undefined, and then set the default value to `assignedUnitRole` using "setState".

This should leave the "render" implementation untouched.